ResearchKit/Common/ORKSkin.m

```diff
@@ -236,7 +236,13 @@ static ORKScreenType ORKGetHorizontalScreenTypeForBounds(CGRect bounds) {
         // Use this method instead of UIApplication's keyWindow or UIApplication's delegate's window
         // because we may need the window before the keyWindow is set (e.g., if a view controller
         // loads programmatically on the app delegate to be assigned as the root view controller)
-        window = [UIApplication sharedApplication].windows.firstObject;
+        
+        for (UIScene *scene in UIApplication.sharedApplication.connectedScenes) {
+            if ([scene.delegate conformsToProtocol:@protocol(UIWindowSceneDelegate)]) {
+                window = [(id<UIWindowSceneDelegate>)scene.delegate window];
```

Comment:
Do you want to add a break  if you get into this condition?
Rather than relying on there only being one connected scene.